dips/dip-10.md

```diff
@@ -13,47 +14,37 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+Diem ID provides a convenient way to identify a user's account within a VASP.
+* For peer-2-peer payments, Diem ID allows a user to provide a human-readable ‘identifier’ to the sender. Instead of sending a [dip-5 address](https://github.com/diem/dip/blob/master/dips/dip-5.md) (that ideally needs to be refreshed for each payment), the receiver can just share their Diem ID once. In this case Diem ID plays the role of an email address for payments. Benefits for this use case:
+  * Diem ID provides better privacy compared to using sub-addresses, because PayAddress is not shared on the chain and will always use a single-use identifier for the on-chain ID
   * From a user's perspective, Diem ID is a persistent identifier.  But from the perspective of the chain, Diem IDs do not exist and instead transactions are have a single-use ID to identify them. By contrast, from the viewpoint of both a user and the chain, a sub-address is an ephemeral user identifier that should be recycled frequently to preserve privacy, but appears on-chain and could thus be used to link payments if it isn't recycled upon each usage.
 
-# User story
-Below is an example of using Diem ID domain for transferring money from one user to another. 
-
-* Receiver shares their Diem ID with the sender (outside the scope of this document)
-* Sender enters receiver's Diem ID and other relevant information into their wallet application
-* Sender's wallet attempts to fetch the receiver's profile picture and full name along with a confirmation dialog
-* Sender confirms the identity of the recipient
-* Sender presses ‘send’ button
-* Receiver's account is credited the amount in the transaction
-* Receiver's wallet displays recent transactions and their senders
+# Diem ID Format
+Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no PII on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format for the identifiers follows:
+`[user_identifier]@[vasp_domain_identifier]`
 
-# Diem ID format
-Diem ID is a string in format `<user-id>$<diem-id-domain>`.
+Example: `alice@avasp`
 
-Example: `andrey$novi`
-
-* **user-id**, identifies an account within a VASP:
+* `user_identifier` is a unique identifier meaningful only to a VASP that represents either the source or destination of a transaction. In practice, they are likely to be re-used and should be capable of both being the sender and recipient in a transaction. Their specific lifetime is undefined but should ideally be meaningful to the owner of the identifier. Specification:
   * Case insensitive
-  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-_+.]*$`
+  * Valid regular expression: ^[a-zA-Z0-9][a-zA-Z0-9-_\.]*$
```

Comment:
I regret removing the +, Adeniyi reminded me an important use case, where I can use +... to indicate a memo line not sure if we want that, but worth poking @aylinaykol